drivers/sx127x/sx127x.c

```diff
@@ -95,17 +101,17 @@ int sx127x_reset(const sx127x_t *dev)
     gpio_set(dev->params.rx_switch_pin);
 #endif
 
-    /* Set reset pin to 0 */
-    gpio_clear(dev->params.reset_pin);
+    /* set reset pin to the state that triggers manual reset */
+    gpio_write(dev->params.reset_pin, SX127X_POR_ACTIVE_LOGIC_LEVEL);
 
-    /* Wait 1 ms */
-    xtimer_usleep(1000);
+    /* Wait for at least 100 µs */
+    xtimer_usleep(110);
```

Comment:
Is the extra 10us delay really needed ? (Same question for other delays below)

Reply:
not strictly. My reasoning here was: the datasheet says 'at least'. For the manual reset before there was 1 ms sleep instead of the specified 'at least 100 µs' and a 10 ms sleep instead of 'at least 5 ms' - effectively multiplying by 10 and doubling what the datasheet is specifying. I didn't want to arbitrarily multiply the time but rather add a reasonable safety margin to ensure that the sleep is enough. I defined 'reasonable' by assuming that even a very bad oscillator should always stay within 10 percent. I'd also be happy to decrease this a little bit - but for a default value I'd rather stick to "better safe than sorry". In the future we should use some kind of board/cpu param to provide information on (worst case) timer/oscillator acurracy.